src/calendar/header/index.js

```diff
@@ -79,14 +79,15 @@ class CalendarHeader extends Component {
     let leftArrow = <View />;
     let rightArrow = <View />;
     let weekDaysNames = weekDayNames(this.props.firstDay);
+    const {testID} = this.props;
 
     if (!this.props.hideArrows) {
       leftArrow = (
         <TouchableOpacity
           onPress={this.onPressLeft}
           style={this.style.arrow}
           hitSlop={{left: 20, right: 20, top: 20, bottom: 20}}
-          testID={CHANGE_MONTH_LEFT_ARROW}
+          testID={testID ? `${this.testID}-${CHANGE_MONTH_LEFT_ARROW}`: CHANGE_MONTH_LEFT_ARROW}
```

Comment:
is `this.testID` contain anything? why `this`?

Reply:
Fixed.